scripts/setup_linux.sh

```diff
@@ -215,46 +202,50 @@ if [[ "${?}" -ne 0 ]]; then
 fi
 
 # Extras
-source activate garage
+conda activate garage
 # Prevent pip from complaining about available upgrades
 pip install --upgrade pip
 # TensorFlow is not in environment.yml because of CPU vs GPU
 # Remove any TensorFlow installations before installing the desired flavor
 pip uninstall -y tensorflow tensorflow-gpu
 if [[ "${_arg_tf_gpu}" = on ]]; then
-  pip install tensorflow-gpu
+  pip install "tensorflow-gpu<1.10,>=1.9.0"
 else
-  pip install tensorflow
+  pip install "tensorflow<1.10,>=1.9.0"
 fi
 # Fix Box2D install
```

Comment:
Can you split each of these items into blocks with a newline? It's hard to tell which steps are related.